api/src/configuration.ts

```diff
@@ -0,0 +1,18 @@
+import { nbVal, strVal } from "@paroi/data-formatters-lib";
+import dotenv from "dotenv";
+
+dotenv.config();
+
+export const appConfig = {
+  thisServer: {
+    port: nbVal(process.env.PORT),
```

Comment:
```suggestion
    port: Number(process.env.PORT),
```
n'aurait pas suffit ?

Reply:
Ça va retourner un NaN si la variable d'environnement n'existe pas

Reply:
@LewisYann